src/Microsoft.ML.ImageAnalytics/ImageLoader.cs

```diff
@@ -196,7 +196,8 @@ protected override Delegate MakeGetter(DataViewRow input, int iinfo, Func<int, b
                                 string path = src.ToString();
                                 if (!string.IsNullOrWhiteSpace(_parent.ImageFolder))
                                     path = Path.Combine(_parent.ImageFolder, path);
-                                dst = new Bitmap(path);
+
+                                dst = new Bitmap(path) { Tag = path };
                             }
                             catch (Exception)
```

Comment:
>catch (Exception) [](start = 28, length = 17)

Why not catch both argument and filenotfound then?  Catching any exception seems dangerous, moreover, you are not capturing inner exception.

Reply:
I'm sorry but what you are talking about does not seem related to my change. I haven't made any change regarding catching exception...